opencv.spec

```diff
@@ -0,0 +1,37 @@
+### RPM external opencv 4.3.0
+## INITENV +PATH PYTHON27PATH %{i}/${PYTHON_LIB_SITE_PACKAGES}
+## INITENV +PATH PYTHON3PATH %{i}/${PYTHON3_LIB_SITE_PACKAGES}
+%define tag 01b2c5a77ca6dbef3baef24ebc0a5984579231d9
```

Comment:
can we not use a traditionally tagged release version?

Reply:
+1
01b2c5a77ca6dbef3baef24ebc0a5984579231d9 anyway points to tag 4.3.0. So we can just use `%{realversion}` instead of tag here.

Reply:
done